webapp/documentation/test_scenarios.md

```diff
@@ -0,0 +1,24 @@
+# Navbar
+  - Signed in state
+    - [ ] Auth: Nav bar should contain user auth (profile photo, name, and sign out). Sign out should kick the user to the authentication page.
+    - [ ] User presence: when another user concurrently uses nook, the person indicator (colored dots) appears. Clicking on them should take to the respective conversation.
+        - Test by adding a conversation id and the time stamp under `user_presence > user email`
+    - [ ] Conversation list panel: A dropdown appears when there are more than 1 conversation shards present.
+	- Test by adding a conversation name in firebase under `nook_conversation_shards > shard-n > name (field), conversations (collection)`
```

Comment:
There's a few more steps to do here, as the number of shards is given by a field the first:

```suggestion
	- If the test project doesn't already have 2 shards, test by setting the field `nook_conversation_shards > shard-0 > num_shards: 2` and adding a second conversation shard with a name `nook_conversation_shards > shard-1 > name: "Conversation list #2"`
```